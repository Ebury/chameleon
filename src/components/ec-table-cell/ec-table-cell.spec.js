import { mount } from '@vue/test-utils';
import EcTableCell from './ec-table-cell.vue';

describe('EcTableCell', () => {
  it('should render as expected', () => {
    const wrapper = mount(
      EcTableCell,
    );

    expect(wrapper.element).toMatchSnapshot();
  });

  it('should use the slot when given content', () => {
    const wrapper = mount(
      EcTableCell,
      {
        slots: {
          default: '<p><a href="#">PO1234A</a<p><p>This is a payment reference</p>',
        },
      },
    );

    expect(wrapper.element).toMatchSnapshot();
  });
});
