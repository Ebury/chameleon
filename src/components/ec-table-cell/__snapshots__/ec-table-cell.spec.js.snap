// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EcTableCell should render as expected 1`] = `
<td
  class="ec-table-cell"
/>
`;

exports[`EcTableCell should use the slot when given content 1`] = `
<td
  class="ec-table-cell"
>
  <p>
    <a
      href="#"
    >
      PO1234A
    </a>
  </p>
  <p>
    This is a payment reference
  </p>
</td>
`;
